tests/test_plugins.py

```diff
@@ -21,21 +24,50 @@
     "dependencies": [],
 }
 
+TEST_PLUGIN_SCRIPT = """
+import asyncio
+import numpy as np
+from imjoy import api
+
+class ImJoyPlugin():
+    def setup(self):
+        api.log('initialized')
+
+    async def run(self, ctx):
+        await api.alert('Hello')
+        await api.log('done')
+
+    def get_array(self):
+        array = np.zeros(10)
+        # array[3] = 93834.356
```

Comment:
There is a strange behavior here, If I enable this line, the engine won't quit after the test is done, I have no idea why.